modules/ROOT/pages/deprecations.adoc

```diff
@@ -1,60 +1,30 @@
 
 
-== Why have we deprecated the GraphQL Architect Graph App? 
-
-The main purpose of the Neo4j GraphQL Architect was to provide users with a Low-Code way to build GraphQL APIs powered by Neo4j. 
-GraphQL Architect was a Graph App for Neo4j Desktop that enabled developers to build, query, and deploy GraphQL APIs backed by the Neo4j graph database, all from within Neo4j Desktop.
-
-We decided it was time to give this tool a new purpose and expand its potential to attract our target audience. 
-Successor of GraphQL Architect Graph App, the new Neo4j GraphQL Toolbox (the name is a placeholder) will be part of the GraphQL for Neo4j holistic product offering. 
-It is currently available at https://graphql-toolbox.neo4j.io/, however we haven't publicized it yet and we're still writing documentation and marketing pieces on it. 
-
-The vision for this is to build an embeddable tool that we can integrate into the Neo4j workspace which will help onboarding developers into GraphQL. 
-It's meant to be a Low-Code GraphQL solution to get started for developers who are either new to Neo4j, or GraphQL, or both, and they're seeking a way to get started easily and quickly with their project from scratch. 
-
-We want to make it easily configurable so that it's at developers' fingertips ready and available for users without requiring them to have credentials to access a live Neo4j instance when they are in a prototyping stage.
-It's also a great way to discover the GraphQL library capabilities. It's very useful for debugging errors and problems users may encounter using the library so they can use the GraphQL Toolbox to reproduce the problem in GitHub issues.
-
-We can communicate to the developers market that there is no mismatch between the API model and the data model (in the database).
-We are graph from top to bottom.
-GraphQL developers already think in graphs about their application data and features, but we need to help them understand that they can store that information in graphs (using a graph DB).  
-
-
-== What has happened to the GraphQL Architect?
+== Neo4j GraphQL Architect
 
-From a technical perspective we have deprecated and removed the GraphQL Architect Graph App from npm and the https://install.graphapp.io/[Graph App gallery], it is no longer available to be installed in Neo4j Desktop.
-The associated GitHub repositories have been archived.
+Neo4j GraphQL Architect was a low-code graph app for Neo4j Desktop that enabled developers to build, query, and deploy GraphQL APIs backed by the Neo4j graph database.
+It was deprecated and removed from npm, the https://install.graphapp.io/[Graph App gallery], and Neo4j Desktop.
+The associated GitHub repositories have also been archived.
 
+This tool was repurposed and expanded with xref:getting-started/toolbox.adoc[Neo4j GraphQL Toolbox], an onboarding, embeddable low-code tool that can be integrated to Neo4j.
```

Comment:
This makes it sound like it was a rewrite, so I would maybe choose different wording here. Also remove "embeddable" again.

```suggestion
This tool was reimagined in the xref:getting-started/toolbox.adoc[Neo4j GraphQL Toolbox], an onboarding, low-code tool that can be integrated to Neo4j.
```